### PR TITLE
Javascript light bulb

### PR DESCRIPTION
**Pull request by Eugenepark215**
Screen Shot 2022-09-06 at 7 12 33 PM
Screen Shot 2022-09-06 at 7 12 37 PM

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+var click = 0;
+var room = document.querySelector('div');
+var lightBulb = document.querySelector('button');
+
+function turnOnOff(event) {
+  click++;
+  if (click % 2 === 0) {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:7]: To determine whether to turn the light on or off, you are making a counter, but the only thing you ever care about is if the counter's value was even or odd. This is a situation where you would be better off using a boolean to keep track of if the light is currently on.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-light-bulb/main.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 var lightBulb = document.querySelector('button');
 
 function turnOnOff(event) {
-  click++;
-  if (click % 2 === 0) {
-    lightBulb.className = 'lightbulb-off';
-    room.className = 'off';
-  } else {
+
+  if (lightBulb.className === 'lightbulb-off') {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:7]: Now your logic is dependent on the state of the DOM to know what to do, which is less than ideal. You should follow my initial suggestion of using a variable containing a boolean value to keep track of if the light is currently on or not, and use that variable's value to decide what action to take when the lightbulb is clicked.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Eugenepark215** [comment]
In progress

**Comment by Eugenepark215** [comment]
Ready for review

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**